cdk/kraken/src/pypi.ts

```diff
@@ -89,9 +89,10 @@ export class PyPIPublishStack extends Stack {
       steps: [
         {
           name: 'Verify tag',
+          shell: 'bash',
```

Comment:
Out of curiosity, why did we define bash here? I thought the original default shell for `ubuntu-latest` is already bash :0

Reply:
Yeah, that's what I thought as well. But it looks like when using a container, GH Actions defaults to sh instead of bash and sh doesn't support the environment variable substitution I was doing.

[Here's the latest run of django-rest-live](https://github.com/pennlabs/django-rest-live/runs/4368410676?check_suite_focus=true) that shows the error message "/__w/_temp/5bd79546-2077-4a31-ab7e-7c4f9264f2d8.sh: 1: Bad substitution"